README.md

```diff
@@ -148,6 +148,7 @@ Safe Logging can be found at [github.com/palantir/safe-logging](https://github.c
     ```
 - `ShutdownHook`: Applications should not use `Runtime#addShutdownHook`.
 - `SwitchStatementDefaultCase`: Switch statements should avoid using default cases. Default cases prevent the [MissingCasesInEnumSwitch](http://errorprone.info/bugpattern/MissingCasesInEnumSwitch.html) check from detecting when an enum value is not explicitly handled. This check is important to help avoid incorrect behavior when new enum values are introduced.
+- `MultimapKeys`: `Multimap.keys()` is usually an error, where the author meant to use `Multimap.keySet()`.
```

Comment:
Don't think it's really good enough to say "foo is usually an error", as that is true by definition for all the checks - this sentence needs to explain to people _why_ we consider this a foot gun!